src/libraries/System.Console/src/System.Console.csproj

```
@@ -164,7 +174,7 @@
     </Compile>
   </ItemGroup>
   <!-- Unix -->
-  <ItemGroup Condition=" '$(TargetsUnix)' == 'true'">
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' AND '$(TargetsiOS)' != 'true'">
```

this disables a lot of other code as well like Interop.Open.cs, is that intentional?

---

also convention is to lower-case and/or in msbuild conditions.

```suggestion
  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' and '$(TargetsiOS)' != 'true'">
```

---

yeah all this stuff is not public and not needed for current impl